Task/Runner/Nuke/Build.cs

```diff
@@ -178,6 +178,42 @@ public class Build : NukeBuild
             }
         });
 
+    Target TestServerE2e => _ => _
```

Comment:
As per my other comment, sorry I gave you the wrong name originally. It should be TestE2e